src/platform/packages/shared/response-ops/rule_form/src/rule_definition/dashboard_service.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { DashboardStart } from '@kbn/dashboard-plugin/public';
+import type { DashboardAttributes } from '@kbn/dashboard-plugin/server';
+
+export interface DashboardItem {
+  id: string;
+  attributes: Pick<DashboardAttributes, 'title'>;
+}
+
+export type DashboardService = ReturnType<typeof dashboardServiceProvider>;
+export type DashboardItems = Awaited<ReturnType<DashboardService['fetchDashboards']>>;
+
+export function dashboardServiceProvider(dashboardService: DashboardStart) {
```

Comment:
@ResponseOps people @dominiqueclarke is working on a related dashboards API in this [PR](https://github.com/elastic/kibana/pull/213287), which will eventually consist of automatically suggested dashboards and manually linked dashboards. Once her PR is merged, I will extend it with the linked dashboards api functionality and eventually the whole `dashboard_service` file will be removed.